tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -325,6 +325,9 @@ func (l *<lexer.name>) Sempred(localctx antlr.RuleContext, ruleIndex, predIndex
  */
 RuleActionFunction(r, actions) ::= <<
 func (l *<lexer.name>) <r.name; format="cap">_Action(localctx <if(r.factory.grammar.lexer)>antlr.RuleContext<else>*<r.ctxType><endif>, actionIndex int) {
+	this := l
+	_ = this
```

Comment:
Assignment to `_` is needed so that Go does not complain if `this` does not end up used at the end.